src/services/style-drawer.ts

```diff
@@ -106,9 +106,9 @@ export const getInputStyles = (
   },
   inputWrapper: {
     // State-conditional background color
-    ...(isFocus
-      ? getStylePropOrDefault(styleProps, 'backgroundColor', TyroPayStylePropKeys.inputFocusBackgroundColor, '#f2f2f2')
-      : getStylePropOrDefault(styleProps, 'backgroundColor', TyroPayStylePropKeys.inputBackgroundColor, '#f9f9f9')),
+    ...getStylePropOrDefault(styleProps, 'backgroundColor', TyroPayStylePropKeys.inputBackgroundColor, '#f9f9f9'),
+    ...(isFocus &&
+      getStylePropOrDefault(styleProps, 'backgroundColor', TyroPayStylePropKeys.inputFocusBackgroundColor, undefined)),
```

Comment:
mirror from iframe, by default the inputFocusBackgroundColor is the same as the inputBackgroundColor